scripts_dev/play_generated_games.py

```diff
@@ -104,6 +103,10 @@ def main():
         agent.reset(env)
 
         if args.vizu >= 0:
+            try:
+                from textworld.envs.wrappers import HtmlViewer
```

Comment:
I think the error would be better handled inside textworld/envs/wrappers/viewer.py. That way we would still be safe if someone directly imports HtmlViewer (or other modules that require visualization) from their code.

Reply:
done! So now instead of erroring out you'll get a printed message about not having installed the visualization requirements.

Reply:
But it wil fail later? I think it is better to fail on the import of HtmlViewer. Like you ve done im render.py.